src/main/java/bio/terra/service/resourcemanagement/google/GoogleProjectService.java

```diff
@@ -152,6 +152,7 @@ private GoogleProjectResource newProject(
                 .setName(requestedProjectId)
                 .setProjectId(requestedProjectId)
                 .setParent(parentResource);
+        logger.info("creating project with request: {}", requestBody);
```

Comment:
```suggestion
        logger.info("creating project `{}` with request: {}", requestBody.toPrettyString(), requestBody);
```

Maybe use `toPrettyString`?

Reply:
Though I will point out that `.toPrettyString()` throws an `IOException` :(

Reply:
Just checked and both toString and toPrettyString seem to print the same thing